src/components/App/App.js

```diff
@@ -1,14 +1,25 @@
 import './App.css';
 import { useDispatch, useSelector } from 'react-redux';
+import { addFriend } from '../../actions';
+import React, { useState, useEffect } from 'react';
 import AddFriend from '../AddFriend/AddFriend';
 import Friends from '../Friends/Friends';
 import Header from '../Header/Header';
 import UpcomingBirthdays from '../UpcomingBirthdays/UpcomingBirthdays';
+import { mockData } from '../../mockData';
+
+const App = () => {
+  const dispatch = useDispatch();
+  const [user, setUser] = useState([]);
+
+  useEffect(() => {
+    setUser(mockData.data.attributes);
```

Comment:
Similar to the other comment, I wonder if we should just store the user name in state, since we aren't displaying the other attributes anywhere